src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,69 @@
+.stopwatch {
+  margin: 0;
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: center;
+  height: $stopwatch-height;
+  width: $stopwatch-width;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    display: block;
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $minutes-hand-width;
+    top: $minutes-hand-top;
+    left: $minutes-hand-left;
+    background-color: #0700ff;
+    transform-origin: bottom;
+    transform: $transform;
+    animation: rotate 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    display: block;
+    position: absolute;
+    height: $seconds-hand-height;
+    width: $seconds-hand-width;
+    top: $seconds-hand-top;
+    left: $minutes-hand-left;
+    background-color: #2c8000;
+    transform-origin: bottom;
+    transform: $transform;
+    animation: rotate 60s linear infinite;
+  }
+
+  &__circle {
+    display: block;
+    position: absolute;
+    height: $circle-height;
+    width: $circle-width;
+    top: $circle-top;
+    left: $circle-left;
+    transform: translate(0, 50%);
+    border-radius: 50%;
+    background-color: #f6a603;
+  }
+}
+
+.stopwatch--speed-up {
+  .stopwatch__minute-hand {
```

Comment:
There is a typo in the class name `.stopwatch__minute-hand`. It should be `.stopwatch__minutes-hand` to match the class defined in the HTML and earlier in the SCSS file.